harness/determined/pytorch/lightning/_adapter.py

```diff
@@ -315,6 +317,23 @@ def lightning_scheduler_dict_to_det(lrs: dict) -> _LRScheduler:
 
         return ordered_optimizers, lr_schedulers
 
+    def _init_optimizers(self) -> Tuple[List, List]:
+        """
+        Configure and validate optimizers and schedulers in pytorch lightning.
+        This implements the functionality of
+        pytorch_lightning.trainer.optimizers.TrainerOptimizersMixin but circumvents the Trainer
+        initialization requirement.
+        """
+        optimizer_conf = self._pls.lm.configure_optimizers()
+        optimizers, lr_schedulers, _, monitor = TrainerOptimizersMixin._configure_optimizers(
```

Comment:
Ok, I see it there in v1.5.9, but I don't see it in tip-of-master for pytorch lightning.  Am I crazy?

Otherwise this is a big red-flag, that we're depending on internals that are constantly changing

Reply:
traced the commit history in PTL, looks like they just moved this logic to pytorch_lightning.core from pytorch_lightning.trainer (https://github.com/PyTorchLightning/pytorch-lightning/blob/master/pytorch_lightning/core/optimizer.py). 
so i think this change is pretty safe, seeing as their internal logic hasn't changed. future upgrades just need to migrate where the static `_ configure_optimizers` call comes from in PTL.

Reply:
That's not "safe", it means that it's going to be a lot of work to continue to work against multiple versions of PTL.

I suppose we can deal with that later.  I don't see a point in rearchitecting our PTL support today.